src/lib/code-sample/javascript.ts

```diff
@@ -0,0 +1,17 @@
+import { camelCase } from 'change-case'
+
+import type { CodeSampleDefinition } from './schema.js'
+
+export const createJavascriptRequest = (
+  request: CodeSampleDefinition['request'],
+): string => {
+  const parts = request.path.split('/')
+  return `await seam.${parts.map((p) => camelCase(p)).join('.')}(${JSON.stringify(request.parameters)})`
+}
+
+export const createJavascriptResponse = (
+  response: CodeSampleDefinition['response'],
+): string => {
```

Comment:
If needed, we can pass in parts of the blueprint here, e.g., the route, endpoints, etc, in case we need to access more context to generate the code sample.